docs/release-notes/february-2024.md

```diff
@@ -0,0 +1,181 @@
+---
+slug: february-2024
+hide_title: true
+description: >-
+    Release notes include the changes, fixes, and additions in specific versions of Semgrep.
+toc_max_heading_level: 3
+title: February 2024
+tags:
+  - Release notes
+---
+
+# Semgrep release notes for February 2024
+
+## 🔧 OSS Engine
+
+* The following versions of the OSS Engine were released in February 2024:
+  * [<i class="fas fa-external-link fa-xs"></i>1.59.1](https://github.com/semgrep/semgrep/releases/tag/v1.59.1)
+  * [<i class="fas fa-external-link fa-xs"></i>1.60.0](https://github.com/semgrep/semgrep/releases/tag/v1.60.0)
+  * [<i class="fas fa-external-link fa-xs"></i>1.60.1](https://github.com/semgrep/semgrep/releases/tag/v1.60.1)
+  * [<i class="fas fa-external-link fa-xs"></i>1.61.1](https://github.com/semgrep/semgrep/releases/tag/v1.61.1)
+  * [<i class="fas fa-external-link fa-xs"></i>1.62.0](https://github.com/semgrep/semgrep/releases/tag/v1.62.0)
+  * [<i class="fas fa-external-link fa-xs"></i>1.63.0](https://github.com/semgrep/semgrep/releases/tag/v1.63.0)
+
+## 🌐 Cloud Platform
+
+### Added
+
+- API: Added a `rule` object under `findings` with the following fields: 
+    - `name`
+    - `message`
+    - `confidence`
+    - `category`
+    - `subcategories`
+    - `technologies`
+    - `vulnerability_classes`
+    - `cwe_names`
+    - `owasp_names` <!-- 12868 --> 
+- Added distinction between Pro engine and OSS findings in the Playground and Editor. <!-- 12275 -->
+- Added support for the `linux-arm64` platform when you download Semgrep Pro Engine. <!-- 12430 -->
+
+### Changed
+
+- Updated the Semgrep Cloud Platform (SCP) login page. <!-- 12744 -->
+- Updated the login process from the CLI to SCP. This change affects new users. <!-- 12531 -->
+- Updated the Semgrep installation instructions for Docker. <!-- 12531 -->
+- Improved performance of Semgrep Playground and Editor. <!-- 12461 -->
+
+### Fixed
+
+- Fixed a bug where the navigation sidebar covered the entire mobile screen and could not be collapsed. <!-- 12876 -->
+- Scan summary links printed after users run `semgrep ci` now reflect a
+  custom `SEMGREP_APP_URL` if set.
+
+## 💻 Code
+
+### Added
+
+* Support for C and C++ is now generally available (GA).
```

Comment:
Could we specify interfile analysis, proprietary rules, etc? cc @milanwilliams

Reply:
AFAIK we did not add proprietary rules, but I did add that we have both cross-file and cross-function analysis.